### PR TITLE
Handle negative input for range commands

### LINKED ISSUE #121: [PE-D][Tester E] range -p command cannot handle multiple comma input
When either lower or upper range inputs are all comma and user hits enter, nothing happens, no error messages as well

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: therealdaofu/ped#1

### LINKED ISSUE #123: [PE-D][Tester E] range -p command cannot handle alphabet or symbol inputs
Similar to the case when input is just commas, when user inputs any letters or symbols and hits enter, nothing happens and no error message is given.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: therealdaofu/ped#2

### LINKED ISSUE #169: [PE-D][Tester D] `range` prefixes (`l/` and `u/`) accept negative integers, which should not be possible
`range -p l/-1000 u/0` will display nothing, which is correct, but the inputs should not be valid. Perhaps display an error message for this scenario.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: t1mzzz/ped#9

### PR DESCRIPTION
**Pull request by yuehernkang**
Modify range command to reject invalid inputs for the range such as alphabets or negative numbers

closes #169 
closes #121 
closes #123

### CODE REVIEW

**Review by zacchaeuschok — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -305,7 +305,11 @@ Format: `list -c`
 ### Filtering client profiles: `filter -c`
 
 Returns a filtered list of clients according to specified tags.
+
+TAG have to be fully specified (i.e. no partial tags like 'high' instead of 'high-end')
```
Comment (lkwlkww) [inline docs/UserGuide.md:309]: Good spot here by specifying this in the UG. Being able to search using partial tags is a good-to-have feature, but its too late to implement that.

Perhaps an improvement for this PR is to make the title a bit clearer (or do this in a separate PR), since this part has nothing to do with negative inputs for range command

**Review by lkwlkww — APPROVED** [review]
